client/src/pages/dashboard/views/Invites.svelte

```diff
@@ -40,6 +43,12 @@
     function resetContext() {
         ctx = null;
     }
+
+    $: invite = inviteList.load().catch(err => {

```

Comment:
Note that this isn't reactive. The `inviteList` will not react to a `reload` because we have not yet subscribed to the store (typically via the `$` prefix).